english/explore-outside/journey-to-flaming-tower/walk/sailboat/enter/wake/decide-to-help/axe/axe.md

```diff
@@ -0,0 +1,2 @@
+You realize you'd like to fight the beast with a large axe. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.